src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -12,12 +12,6 @@
     <TargetOS Condition="'$(OSGroup)'!='AnyOS'">$(OSGroup)</TargetOS>
   </PropertyGroup>
 
-  <Target Name="RestoreTestRuntimePackage"
```

Comment:
I think it is easier to just update WCF to account for the breaking changes. See https://github.com/dotnet/wcf/compare/master...mellinoe:test-runtime-projectref